templates/docs/examples/patterns/empty-state/error-management.html

```diff
@@ -6,15 +6,18 @@
 {% block content %}
 <section class="p-strip">
   <div class="row">
```

Comment:
It seems that the canonical.com version is using older (the "default") grid class names.

We could utilise the new "shorthand" layouts here.

So instead of having `.row > .col-3.col-medium-2 + .col-9.col-medium-4`, we could use the `.row--25-75 > .col + .col` - that should cover the responsiveness of the grid.

```suggestion
  <div class="row--25-75">
```

Reply:
Made suggested changes to use new grid class names!